Integrations/integration-OTRS.yml

```diff
@@ -0,0 +1,971 @@
+description: Service management suite that comprises ticketing, workflow automation,
+  and notification.
+configuration:
+- display: OTRS Server URL (e.g. https://demisto.managed-otrs.com)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: OTRS Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Trust any certificate (unsecure)
+  name: unsecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Queues to fetch tickets from  ("Any" fetches from all queues. CSV supported,
+    e.g., Misc,Raw)
+  name: fetch_queue
+  defaultvalue: Any
+  type: 0
+  required: false
+- display: Fetch tickets in priority
+  name: fetch_priority
+  defaultvalue: ""
+  type: 16
+  required: false
+  options:
+  - 1VeryLow
+  - 2Low
+  - 3Normal
+  - 4High
+  - 5VeryHigh
+- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3
+    months, 1 year)
+  name: fetch_time
+  defaultvalue: 3 days
+  type: 0
+  required: false
+script:
+  script: |-
+    ''' IMPORTS '''
+    import requests
+    from pyotrs import Article, Client, Ticket, DynamicField, Attachment
+    import base64
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    ''' GLOBAL VARS '''
+
+    SERVER = demisto.params()['server'][:-1] if demisto.params()['server'].endswith('/') else demisto.params()['server']
+    USERNAME = demisto.params()['credentials']['identifier']
+    PASSWORD = demisto.params()['credentials']['password']
+    USE_SSL = not demisto.params().get('unsecure', False)
+    FETCH_QUEUE = demisto.params().get('fetch_queue', 'Any')
+    FETCH_PRIORITY = demisto.params().get('fetch_priority')
+    FETCH_TIME = demisto.params().get('fetch_time', '3 days')
+
+    ''' HELPER FUNCTIONS '''
+
+    def string_time_to_datetime(string_time):
+
+        if string_time == 'Last24Hours':
```

Comment:
why don't we use our time range conventions? `2 days` `3 hours`

Reply:
We do, removed the function.